cmake/CPM.cmake

```diff
@@ -0,0 +1,475 @@
+# CPM.cmake - CMake's missing package manager
```

Comment:
Also do we need this one?

Reply:
You are right, I will remove all cmake related stuff and push a commit.

Reply:
Thanks!